app/app.py

```diff
@@ -1,12 +1,30 @@
-from flask import Flask
+from flask import Flask, jsonify, request
+from os import path
+import subprocess
+import json
+import os
 
 app = Flask(__name__)
 
+# Reading directory path from config.json file
+with open('config.json') as json_data_file:
+    config = json.load(json_data_file)
+    WORKSPACE_DIR = config["path1"]
 
-@app.route('/hello')
-def index():
-    return "Hello, World!"
+# Route for creating workspaces
+@app.route('/workspace/create', methods=['POST'])
+def linchpin_init():
+    data = request.json     # Get request body
+    name = data["name"]
+    cmd = "linchpin init" + " " + name
```

Comment:
This command would not work 
for example if post name as test
the command would be 
```
linchpin init test 
```
which creates an empty folder named test inside the workspace directory. 
try it out. 
What ideally should happen is, 
test directory should be created first and ``` linchpin init ``` should be run inside the directory.